src/data/endpoints/managed.yaml

```diff
@@ -238,6 +238,39 @@ endpoints:
             curl -H "Authorization: Bearer $TOKEN" \
                 -X DELETE
                 https://$api_root/$version/managed/services/$service_id
+  /managed/services/$id/enable:
+    group: Services
+    type: Action
+    authenticated: true
+    description: >
+      Enables a Managed service your account may access.
+    methods:
+      POST:
+        description: >
+          Enables this Managed service. Will only enable if it is currently
+          disabled. An enabled service is actively monitored based on criteria
+          specified in your service setup.
+        examples:
+          curl: |
+            curl -H "Authorization: Bearer $TOKEN" \
+                -X POST
+                https://$api_root/$version/managed/services/$service_id/enable
+  /managed/services/$id/disable:
+    group: Services
+    type: Action
+    authenticated: true
+    description: >
+      Disables a Managed service your account may access.
+    methods:
+      POST:
+        description: >
+          Disables this Managed service. Will only disabled if it is currently
+          enabled. A disabled service will not be actively monitored.
```

Comment:
Maybe link to [this guide](https://www.linode.com/docs/platform/linode-managed/) that has info on disabling a service/steal words from it.  Of note, you say `Will only disabled if` above.